File: packages/implementations/OneLang-Json-v0.1/PHP/src/OneJson.php
<?php

namespace OneLang\Json;

class OneJValue {
    function __construct($value) { $this->value = $value; }

    function asObject() { return new OneJObject($this->value); }
    function asString() { return $this->value; }
    function getArrayItems() { return array_map(function($item) { return new OneJValue($item); }, $this->value); }
}

class OneJObject {
    function __construct($value) { $this->value = $value; }

    function get($key) { return new OneJValue($this->value[$key]); }
}

class OneJson { 
    static function parse($str) {
        return new OneJValue(json_decode($str, TRUE));
    }
}

File: packages/implementations/OneLang-Json-v0.1/Python/src/OneJson.py
import json

class OneJValue:
    def __init__(self, value):
        self.value = value
    
    def as_object(self):
        return OneJObject(self.value)
    
    def as_string(self):
        return self.value
    
    def get_array_items(self):
        return [OneJValue(item) for item in self.value]

class OneJObject:
    def __init__(self, value):
        self.value = value
    
    def get(self, key):
        return OneJValue(self.value[key])

class OneJson:
    @staticmethod
    def parse(content):
        return OneJValue(json.loads(content))